.github/workflows/build.yml

```
@@ -85,7 +85,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [1.52.1]
+        rust: [stable]
```

```suggestion
        rust: [1.60.0]
```
I prefer to keep clippy pinned because the hints change frequently.